Cargo.toml

```diff
@@ -1,22 +1,23 @@
 [package]
 name = "finl_unicode"
-version = "1.2.0"
+version = "1.3.0"
 edition = "2021"
-license = "MIT OR Apache-2.0"
+license = "(MIT OR Apache-2.0) AND Unicode-DFS-2016"
```

Comment:
Looks to me like this should be `Unicode-3.0`. `Unicode-DFS-2016` was an older version of the license (… though the differences are basically just formatting).

I’m also not *certain* how appropriate putting it as a license is. If you include the data files, yes: that particular expression is copyrightable; but if you just integrate the data, I *believe* that’s generally agreed to be uncopyrightable, being just data.